backend-shared/config/db/queries.go

```diff
@@ -199,8 +199,12 @@ type PostgreSQLDatabaseQueries struct {
 	allowUnsafe bool
 }
 
-func NewProductionPostgresDBQueries(verbose bool) (DatabaseQueries, error) {
-	db, err := connectToDatabase(verbose)
+func NewProductionPostgresDBQueries(verbose bool, args ...int) (DatabaseQueries, error) {
```

Comment:
Recommend creating two functions, so that the calling code can decide whether to specify a port, or not:
```golang

func NewProductionPostgresDBQueries(verbose bool) (DatabaseQueries, error) {
    return NewProductionPostgresDBQueriesWithport(verbose, DEFAULT_PORT)
}


func NewProductionPostgresDBQueriesWithPort(verbose bool, port int) (DatabaseQueries, error) {
    // As before
}
```